ddtrace/_trace/trace_handlers.py

```diff
@@ -692,6 +692,12 @@ def _on_botocore_patched_bedrock_api_call_success(ctx, reqid, latency, input_tok
         span.set_metric("bedrock.response.usage.prompt_tokens", int(input_token_count))
     if output_token_count:
         span.set_metric("bedrock.response.usage.completion_tokens", int(output_token_count))
+    usage = {}
+    if input_token_count:
+        usage["input_tokens"] = int(input_token_count)
+    if output_token_count:
+        usage["output_tokens"] = int(output_token_count)
+    ctx.set_item("llmobs.usage", usage)
```

Comment:
Can we move llmobs-specific code out of `trace_handlers.py` and add this information directly to the `TracedBotocoreStreamingBody` in `services/bedrock.py` instead?